src/components/form/examples/form-with-help.tsx

```diff
@@ -0,0 +1,49 @@
+import { Component, h, State } from '@stencil/core';
+import { schema } from './help-form-schema';
+
+/**
+ * Form elements with help
+ * It's possible to add extensive help to any form element.
+ * The string you provide can be in Markdown format,
+ * empowering you present a rich-text experience to the user,
+ * including bullet points, hyperlinks, etc…
+ * Read more on [Help](#/component/limel-help) component.
+ *
+ * :::note
+ * This "help" is not the same as providing a so called "helper text".
+ * Helper text is a short description of the input fields
+ * that becomes visible when the user click on the fields.
+ * It is most often used to render validation errors, or provide
+ * brief clues about the filed or its expected value.
+ *
+ * Check out the [Input field Component](#/component/limel-input-field)
+ * examples, where we describe how to properly use `helperText`, and `placeholder`.
+ *
+ * In the context of forms, helper texts are also used to
+ * provide validation feedback to the user, about an invalid input.
```

Comment:
This can be confusing. 

"In the context of forms" is vague. I would say the helper text is only ever used in the context of forms! But you mean when rendering a `limel-form` using a schema. Which leads to the question, how do you set `helperText` then? I couldn't find this explained anywhere, but in some examples we set `description` for a field in the schema which is used as helper text.

I think it would be helpful to clearly explain that limel-form will either pass a validation error as helper text, or the schema description when there's no error.

(maybe this example is not the right place for documenting helper texts in detail, but we currently don't have any other place for how they work in limel-form)

Reply:
my intention was mostly providing mental help for decision making, like when to use `helperText`, `placeholder`, or `help`.

> I think it would be helpful to clearly explain that limel-form will either pass a validation error as helper text, or the schema description when there's no error.

But how would you explain this one? Can you add a suggestion please.

Reply:
```suggestion
 * When rendering a form with a schema, helper texts are automatically passed for all the fields based on the schema and validation errors. The schema `description` for a field is used when there are no errors, otherwise the first validation error is displayed in the place of the helper text.
```

Reply:
This description is not completely accurate though, because the `isInvalid` check is a bit more complicated than that:
https://github.com/Lundalogik/lime-elements/pull/2601

I don't think the consumer needs to be bothered with the exact details of when errors are shown, but maybe this is better:

"The schema `description` for a field is used as helper text while the field is shown as valid. When there is validation feedback to provide to the user, the field is instead marked as invalid with an error message that is displayed in the place of the helper text."